spdx/writers/tagvalue.py

```diff
@@ -343,19 +341,27 @@ def write_document(document, out, validate=True):
     write_creation_info(document.creation_info, out)
     write_separators(out)
 
-    # Writesorted reviews
-    for review in sorted(document.reviews):
-        write_review(review, out)
-        write_separators(out)
+    # Write sorted reviews
+    if document.reviews:
+        out.write("# Reviews\n\n")
+        for review in sorted(document.reviews):
+            write_review(review, out)
+            write_separator(out)
+        write_separator(out)
```

Comment:
Reviews and annotations use `write_separator` at the end while all other entries finish with `write_separators`.

Sorry I didn't catch this in the first round!

Reply:
This is intended but I also missed this for the extracted licenses. If I would use `write_separators` there, I would get 3 free lines between reviews and the next block which would be annotations. So to be consistent I use `write_separator` here and get 2 free lines between each block.

Reply:
Ah, of course. All right then :)